scripts/ci-build-and-test

```diff
@@ -75,6 +75,10 @@ fi
 
 mkdir ${PWD}/destdir
 ${MAKE} install DESTDIR=${PWD}/destdir || exit $?
+# Query value of ${CMAKE_INSTALL_PREFIX} from CMake cache (https://stackoverflow.com/questions/8474753/how-to-get-a-cmake-variable-from-the-command-line)
+# Remove drive letter, which might precede the path on Windows (MinGW) platform.
+INSTALL_PREFIX=`cmake -L -N . | grep CMAKE_INSTALL_PREFIX | cut -d "=" -f2 | cut -d ":" -f2`
```

Comment:
Why do you overwrite the value of `INSTALL_PREFIX` here?

Reply:
In the MinGW environment, CMake seems to automatically adjust the `CMAKE_INSTALL_PREFIX`. Even though CMake is being called as follows:

```
cmake -DCMAKE_INSTALL_PREFIX=${INSTALL_PREFIX:=/usr/local} ...
```

the effective value of `CMAKE_INSTALL_PREFIX` turns out to be `C:/tools/msys64/usr/local`. You can observe this in the [log output](https://travis-ci.com/github/stukowski/kim-api/builds/200086002#L606).

The purpose of the code I've added is to query the actual value of the `CMAKE_INSTALL_PREFIX` variable and use in the following lines to compute the install path. It's not strictly necessary to (re-)use the shell variable name "`INSTALL_PREFIX`" for this. But I thought it makes sense. On Unix/Linux systems, it will already have the same value as `CMAKE_INSTALL_PREFIX` and overwriting it shouldn't hurt.